docs/examples-gcp-gke.md

```diff
@@ -0,0 +1,101 @@
+# GCP-GKE
+
+Cluster.dev uses [stack templates](https://docs.cluster.dev/stack-templates-overview/) to generate users' projects in a desired cloud. GCP-GKE is a stack template that creates and provisions Kubernetes clusters in GCP cloud by means of Google Kubernetes Engine (GKE).
+
+In this repository you will find all information and samples necessary to start an GKE cluster on GPC with Cluster.dev. 
```

Comment:
On this page you will find all information and samples necessary to start an GKE cluster on GPC with Cluster.dev.